Misc/NEWS.d/next/Library/2024-08-27-18-58-01.gh-issue-123401.t4-FpI.rst

```diff
@@ -0,0 +1,2 @@
+The :mod:`http.cookies` module now supports parsing obsolete :rfc:`850`
+date formats, in accordance with :rfc:`9110` requirements.
```

Comment:
You can credit yourself if you want. Add something like `Patch by YOUR_NAME.` on a separate line.

Reply:
```suggestion
date formats, in accordance with :rfc:`9110` requirements.
Patch by Nano Zheng.
```

Reply:
Am I doing this correctly?

Reply:
Yes (actually, you can just look at other files and copy how they do it).